### DISCUSSION TITLE
Failed to start gauge API: Error occurred while starting runner process.

### DISCUSSION THREAD

**Discussion by jnguyen617** [getgauge/gauge#2245] [General]
Hi, I ran into an issue with running example.spec and couldn't figure what went wrong. Does anyone ran into this issue?

jtest3 % npm test

> test
> gauge run specs/

Error ----------------------------------

[Gauge]
Failed to start gauge API: Error occurred while starting runner process.
Error : fork/exec ./index.js: permission denied

Get Support ----------------------------
	Docs:          https://docs.gauge.org
	Bugs:          https://github.com/getgauge/gauge/issues
	Chat:          https://github.com/getgauge/gauge/discussions

Your Environment Information -----------
	darwin, 1.4.3, f98dd40
	html-report (4.1.4), js (2.4.0), screenshot (0.1.0)
john.n@C02F97X0MD6M jtest3 % gauge run specs
Error ----------------------------------

[Gauge]
Failed to start gauge API: Error occurred while starting runner process.
Error : fork/exec ./index.js: permission denied

Get Support ----------------------------
	Docs:          https://docs.gauge.org
	Bugs:          https://github.com/getgauge/gauge/issues
	Chat:          https://github.com/getgauge/gauge/discussions

Your Environment Information -----------
	darwin, 1.4.3, f98dd40
	html-report (4.1.4), js (2.4.0), screenshot (0.1.0)
john.n@C02F97X0MD6M jtest3 %

**Comment by sriv** [comment]
` fork/exec ./index.js: permission denied` -> this seems to be the cause. Can you please check that gauge-js is installed with the right permissions for the user?  Try running `gauge uninstall js` and reinstall using `gauge install js`.

**Reply by jnguyen617** [reply]
It is the permission for .gauge folder. Add wx permission to it and the problem resolved. Thanks
